Sources/App/Models/Repository.swift

```diff
@@ -126,7 +126,7 @@ final class Repository: Model, Content {
     
     init(id: Id? = nil,
          package: Package,
-         authors: [Author] = [],
+         authors: PackageAuthors? = nil, // [Author] = [],
```

Comment:
nit: cleanup the comment here
```suggestion
         authors: PackageAuthors? = nil,
```